format_strawberryfield.routing.yml

```diff
@@ -45,7 +45,7 @@ format_strawberryfield.metadatadisplay_settings:
   path: 'admin/structure/metadatadisplay_entity_settings'
   defaults:
     _form: '\Drupal\format_strawberryfield\Form\MetadataDisplaySettingsForm'
-    _title: 'Metadata Display Settings'
+    _title: 'Metadata display settings'
```

Comment:
I just noticed this. Is it some Drupal convention to use names in lower case?

Reply:
I noticed the "core" ones are that way, just my OCD kicking in

Reply:
Will make an habit of that. Thanks!